### PR TITLE
[compliance] landuse_classification

### PR DESCRIPTION
**Pull request by sandhujasmine**
- updated lock file and removed pins for now from anaconda-project.yml
- updated catalog.yml so data loads as before
- small updates to calls since so warnings are fixed (API likely changed w/ updated packages)

### CODE REVIEW

File: landuse_classification/Image_Classification.ipynb [hunk 1]
```diff
@@ -63,7 +62,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
```
Comment (jbednar) [inline landuse_classification/Image_Classification.ipynb:66]: These details should get cleared by the kernel stripping script.

File: landuse_classification/anaconda-project.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ description: Image classification using the UC Merced Land Use Dataset
 examples_config:
   maintainers:
   - jbednar
+  - philippjfr
```
Comment (jbednar) [inline landuse_classification/anaconda-project.yml:7]: Philipp did way more on this than I did:
```suggestion
  - philippjfr
  - jbednar
```

File: landuse_classification/Image_Classification.ipynb (hunk 1 above)
Comment (sandhujasmine) [reply landuse_classification/Image_Classification.ipynb]: Fixed!

**Review by maximlt — APPROVED** [review]
We did it @sandhujasmine !!! Thanks for your work, that helped me a lot to complete this PR.

--

This is not a modernization kind of PR, it's just to make this project valid and allow it to be tested and built on the CI. Talking about the build, fitting the model takes an hour on the CI, it's long compared to most examples but still reasonable (10 hours wouldn't be). One issue I observed is that the build is not deterministic, during the last one the model performed really badly and I have to re-run the build, which fortunately gave some decent result (I set the seed previously but it's not enough apparently). Not a blocker for this PR to be merged.

**Pull request merged**